hera_mc/tests/test_db_check.py

```diff
@@ -24,7 +24,7 @@ class ValidTestModel(Base):
         __tablename__ = "validity_check_test"
 
         #: Running counter used in foreign key references
-        id = Column(Integer, primary_key=True)
+        id_ = Column(Integer, primary_key=True)
```

Comment:
It looks like in the main code, we add the `#noqa` option, rather than redefine `id`. Is that approach available there as well? No worries if not, I don't think the `#noqa` is actually harmful.

Reply:
Hm interesting. I bet this got changed to not overload the build in `id` function. I am definitely in favor of that, but also understand where you're coming from. I would propose we use a better variable name in general :laughing:

Reply:
probably though this is because psql like the `id` variable name when python doesn't